locations/spiders/accord_bio_fr.py

```
@@ -0,0 +1,56 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+import re
+
+class AccordBioFrSpider(SitemapSpider, StructuredDataSpider):
+    name = "accord_bio_fr"
+    item_attributes = {
+        "brand": "Accord Bio",
+        "brand_wikidata": "Q140930088"
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)